src/components/Results/Results.tsx

```diff
@@ -44,6 +48,15 @@ const Results: React.FC<Props> = ({ results, measureFile, htmls }) => {
             </IconButton>
           )}
         </Grid>
+        {results &&
+          fhirpath.evaluate(results, 'MeasureReport.group').map((group: any) => {
```

Comment:
```suggestion
          fhirpath.evaluate(results, 'MeasureReport.group').map((group: R4.IMeasureReport_Group) => {
```

see above comment regarding stricter typing of this